spec/features/edit_batch_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+describe "edit batch form and find proper date fields", type: :feature do
+  let(:user) do
+    User.create(:username => "test", :group_list => "admin")
+  end
+
+  let(:file_title) { 'test file' }
+  let(:batch) { Batch.create }
+  let(:field_labels) {{'Date Accepted' => '2',
+                       'Date Available' => '3',
+                       'Date Copyrighted' => '4',
+                       'Date Collected' => '5',
+                       'Date Issued' => '6',
+                       'Date Submitted' => '7',
+                       'Date Modified' => '8',
+                       'Date Valid' => '9'}}
+  let(:file) do
+    GenericFile.new(batch: batch) do |f|
+      f.title = [file_title]
+      f.label = "test"
+      f.apply_depositor_metadata(user.user_key)
+      f.save!
+    end
+  end
+
+  before do
+    login_as user, :scope => :user
+    visit "/batches/#{file.batch.id}/edit"
+    page.find('#show_addl_descriptions').click
+  end
+
+  context "when ingesting a file", js:true do
+    it "should display proper date fields" do
```

Comment:
Add something on the end of the string to the effect of "...when the Add Date button is clicked". Could also separate the tests out so it checks that the new date fields don't show by default, and then does another check to make sure they do.